Kickstarter-iOS/Views/Controllers/DiscoveryPageViewController.swift

```diff
@@ -362,17 +362,31 @@ internal final class DiscoveryPageViewController: UITableViewController {
     self.present(vc, animated: true, completion: nil)
   }
 
-  fileprivate func goTo(project: Project, initialPlaylist: [Project], refTag: RefTag) {
-    let vc = ProjectNavigatorViewController.configuredWith(
-      project: project,
-      refTag: refTag,
-      initialPlaylist: initialPlaylist,
-      navigatorDelegate: self
+  fileprivate func goTo(project: Project, initialPlaylist _: [Project], refTag: RefTag) {
+    /** FIXME: This is the only entry point to the new project page, as we build it out and get it fully functional we'll need to refactor entry points in `ProjectNavigatorPagesDataSource` and `ProjectNavigatorViewController` (basically search for places where `ProjectPamphletViewController` is used in the app.)
+     let vc = ProjectNavigatorViewController.configuredWith(
+       project: project,
+       refTag: refTag,
+       initialPlaylist: initialPlaylist,
+       navigatorDelegate: self
+     )
+
+     if UIDevice.current.userInterfaceIdiom == .pad {
+        vc.modalPresentationStyle = .fullScreen
+     }
+
+     self.present(vc, animated: true, completion: nil)
+     */
+
+    let projectParam = Either<Project, Param>(left: project)
```

Comment:
Could you explain why you're using `Either` here? Looks like you are initializing it with the left and the right is not set up to do anything in particular? 

My understanding of this is that there is just one entry in and a `Project` is absolutely necessary to set up the views.

Reply:
Ah it looks like this is the original implementation based on some code below?